src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/BerConverter.cs

```diff
@@ -37,7 +37,7 @@ public static byte[] Encode(string format, params object[] value)
                 if (fmt == '{' || fmt == '}' || fmt == '[' || fmt == ']' || fmt == 'n')
                 {
                     // no argument needed
-                    error = Wldap32.ber_printf_emptyarg(berElement, new string(fmt, 1));
+                    error = NativePal.BerPrintfEmptyarg(berElement, new string(fmt, 1));
```

Comment:
>NativePal [](start = 28, length = 9)

don't we call this Interop instead?

Reply:
sure, I can rename to  Interop as well.

Reply:
I'd advise against naming it Interop.  Interop is the name of the root type that we put library specific P/Invokes under.  (e.g. `internal static class Interop { `internal static class crypt32 { ... } }`).  Since this is hybridizing across different libraries, a Pal name is better.  Though I'd advise naming it for purposes, rather than "Native".  e.g. `BerPal.PrintfEmptyArg`

Reply:
Ok, I'll name it LdapPal in order to be able to put both ldap and ber calls in there.